src/registrar/tests/test_admin.py

```diff
@@ -36,6 +37,20 @@
 logger = logging.getLogger(__name__)
 
 
+class TestDomainApplicationAdminForm(TestCase):
+    def setUp(self):
+        # Create a test application with an initial state of started
+        self.application = completed_application()
+
+    def test_form_choices(self):
```

Comment:
(Optional) I'd add a second test here (largely copy + paste of this first one) with an invalid/weird value for choices (form.fields["status"].widget.choices = None or []). Maybe we expect an exception or just a straight return. This would indirectly test the actual model in addition to expanding the breadth of this test

Reply:
i added this test; the expected return is the full list of choices from the Domain Application model though, in order to successfully handle the case of adding a new Domain Application through django admin

Reply:
Given your comment about the FSM, I think that's very reasonable. Nice addition!